src/components/FamilyManifestModal/FamilyManifestModal.js

```diff
@@ -207,38 +207,41 @@ export default compose(
             ));
           };
           const FooterWithParticipantsOnly = createFooterComponent(participantIds);
-          return (
+          const participantSection = (
+            <Section>
+              <ModalSubHeader className={`modalSubHeader`}>
+                <span className={`highlight`}>
+                  <Trans>Participants Summary</Trans>
+                </span>
+                <span>
+                  {' '}
+                  <Trans>- all files will be included in the manifest</Trans>.
+                </span>
+              </ModalSubHeader>
+              <Table
+                {...{
+                  reverseColor: !isFamilyMemberFilesAvailable,
+                  stats: [{ icon: null, label: 'Data Types' }, ...participantStats],
+                }}
+              >
+                <ManifestTableDataRow
+                  {...{
+                    fileType: 'All',
+                    members: participantsMemberCount,
+                    files: participantFilesCount,
+                    fileSize: fileSizeToString(participantFilesSize),
+                    isChecked: isFamilyMemberFilesAvailable,
+                    leftComponent: <CheckCircleIcon className={`checkMark`} />,
+                  }}
+                />
+              </Table>
+            </Section>
+          );
+          return !dataTypes ? (
```

Comment:
shouldn't this be driven by a loading state? what if there are no data types?

Reply:
the empty state would come as `[]`, but for better clarity, I've added `loadingProps` flag :)